cmd/destroy/destroy.go

```diff
@@ -539,7 +539,10 @@ func (pc *destroyCommand) waitForNamespaceDestroyAllToComplete(ctx context.Conte
 			case "Active":
 				if hasBeenDestroyingAll {
 					// when status is active again check if all resources have been correctly destroyed
-					cfgList, err := c.CoreV1().ConfigMaps(namespace).List(ctx, metav1.ListOptions{})
+					// list configmaps that belong okteto deployments
+					cfgList, err := c.CoreV1().ConfigMaps(namespace).List(ctx, metav1.ListOptions{
+						LabelSelector: fmt.Sprintf("%s=%s", model.GitDeployLabel, "true"),
```

Comment:
This would only verify for config maps created as part of a develpment environment, we would need to check if any of these labels exists (`dev.okteto.com/git-deploy`, `stack.okteto.com` or `dev.okteto.com/app`). 

I don't remember now if we could check that directly using a label selector. If not, we can list all and iterate over them to check it

Reply:
I went around this and created a resourceLabels map, so once we retrieve the cfgmap items if any of them has one of this labels we return the error, if not, the cfgmaps are not resources one so the error is nil